### DISCUSSION TITLE
What else formating choices can I take (expect epoch, loss and metrics) when using ModelCheckpoint callback  ?

### DISCUSSION THREAD

**Discussion by ZeguanXiao** [Lightning-AI/pytorch-lightning#11384] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Can I get something like working directory in trainer?

**Comment by rohitgr7** [comment]
hey @hainanxzg !

can't really get what you mean here. Would you mind giving more details on what you are trying to do/requesting?

**Reply by ZeguanXiao** [reply]
Sorry. In my case, I use hydra to set a specific running directory for each experiment run. I saved checkpoints of each run in their running directory but the disk space is quickly exhausted. Now I want to save checkpoints to another large space disk and make me clear which checkpoint is belong to which run. 

I want:
/my/path/checkpoint/**run001**/last.ckpt
/my/path/checkpoint/**run002**/last.ckpt

Now I set the dirpath parameter of ModelCheckpoint callback as /my/path/checkpoint, and the checkpoint will override:
/my/path/checkpoint/last.ckpt
